device/mellanox/x86_64-mlnx_msn4600-r0/platform_components.json

```diff
@@ -0,0 +1,14 @@
+{
+    "chassis": {
+        "MSN4600": {
+            "component": {
+                "ONIE": { },
+                "SSD": { },
+                "BIOS": { },
+                "CPLD1": { },
+                "CPLD2": { },
+                "CPLD3": { }
+            }
+        }
+    }
+}
```

Comment:
is this aligned with the HW mgmt definition? we have 3 CPLDs for this system?

Reply:
According to the HW SPEC there are three CPLD devices in the system